titapapunne/MoreThanWords
# Mtw 21

# Overview

The frontend components for a generalizable Outcomes page are included in this PR, as well as backend endpoints to dynamically update Task and Outcome components based on Salesforce data. Frontend infrastructure for querying such endpoints is included as well. Furthermore, this PR includes code for updating Salesforce checkbox fields when the user checks off tasks on the frontend.

In the future, we plan to utilize the authorization token framework to obtain individualized user data so that the values of each checkbox are specific to each user. Currently, our testing only involves Trainee Pod TR-1 and Youth FakeD.

If there are any other questions about the content of this PR, feel free to ask either myself, Julia, or Alec!

# Steps to Test this PR:

1. Load the Expo app on a mobile device.
2. Manually verify that frontend interaction successfully updates Salesforce for example user data.

# Standard Checklist:
